### ISSUE TITLE
api.Gamepad - vibrationActuator unsupported on Chrome Android

### ISSUE THREAD

**Issue by vivadave** [mdn/browser-compat-data#12180]
#### What information was incorrect, unhelpful, or incomplete?

vibrationActuator doesn't work with Chrome Android

#### What did you expect to see?

No support

#### Did you test this? If so, how?

Tested with https://gamepad-tester.com/ and with a test app made by me, vibartionActuator works fine with desktop Chrome but not with Android Chrome (vibrationActuator is alwais null)

<details>
<summary>MDN page report details</summary>

* Query: `api.Gamepad`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Gamepad_API
* Report started: 2021-08-25T13:04:36.533Z

</details>

**Comment by tomayac** [comment]
Somewhat related, I wanted to check `hapticActuators` support on Firefox 96.0.1 (64-bit) on macOS, but it doesn't even detect my pretty standard XBox 360 USB wired game pad. (Needless to say, it works in Chrome, so it's not the gamepad.) Is this a known issue? If you can check, @queengooborg, does Firefox detect your gamepad?

**Comment by queengooborg** [comment]
When digging into this, I have found some very interesting results.  TLDR: `vibrationActuator` _is_ supported on Chrome Android, but support differs between controllers.

I only recently got an Xbox controller (I rarely use controllers, but I use a DualSense if I do), so I couldn't fully test this until now.  Using the web app above (which I actually used quite often when repairing broken controllers for a while, great app!), I found the following support for `vibrationActuator`:

- Chrome Desktop on macOS
  - DualSense: No
  - Xbox Wireless: Yes
- Chrome Android
  - DualSense: Yes
  - Xbox: No

I haven't tested either with Safari, or tested `hapticActuators` on Firefox, but it seems that support may be platform-specific...?
